Domain.FSharp/Shipping/Cargo.fs

```diff
@@ -0,0 +1,14 @@
+namespace FSharp.Domain.Shipping.Cargo
+
+open System
+open System.Text.RegularExpressions
+
+type TrackingId(value: Guid) =
+    member this.Value = value
```

Comment:
Use `_` if you're not using `Value` inside `TrackingId`.

```suggestion
    member _.Value = value
```

Reply:
I'll stick with conventions used by the community. It's a valid comment.